style_static_prefs/Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
-name = "static_prefs"
+name = "stylo_static_prefs"
 version = "0.1.0"
-edition = "2021"
 authors = ["The Servo Project Developers"]
+documentation = "https://docs.rs/stylo_static_prefs/"
+description = "Static configuration crate for Stylo CSS engine"
```

Comment:
It's weird to have he same description as `stylo_config`

Reply:
I have updated the `stylo_config` description as this is correct for this crate, but `stylo_config` is not "static"